app/src/main/res/xml/settings.xml

```diff
@@ -41,6 +41,12 @@
         android:layout_height="wrap_content"
         android:title="@string/settings_category_content_presentation">
 
+        <CheckBoxPreference
```

Comment:
to restart the UI on the pref change, I would add something like the below on SettingsFragment:

```
findPreference(PrefUtils.DARK_THEME)?.setOnPreferenceChangeListener { preference, any ->
			activity.finishAffinity()
			startActivity<MainActivity>()
			true
		}
```